apis/risingwave/v1alpha1/risingwave_types.go

```diff
@@ -156,6 +156,22 @@ type RisingWaveComponentGroupTemplate struct {
 	// metadata of the RisingWave's Pods.
 	// +optional
 	Metadata RisingWavePodTemplatePartialObjectMeta `json:"metadata,omitempty"`
+
+	// List of environment variables to set in the container.
+	// Cannot be updated.
+	// +optional
+	// +patchMergeKey=name
+	// +patchStrategy=merge
+	Env []corev1.EnvVar `json:"env,omitempty" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,7,rep,name=env"`
+
+	// List of sources to populate environment variables in the container.
+	// The keys defined within a source must be a C_IDENTIFIER. All invalid keys
+	// will be reported as an event when the container is starting. When a key exists in multiple
+	// sources, the value associated with the last source will take precedence.
+	// Values defined by an Env with a duplicate key will take precedence.
+	// Cannot be updated.
+	// +optional
+	EnvFrom []corev1.EnvFromSource `json:"envFrom,omitempty"`
```

Comment:
Do we currently use `EnvFrom`? Do we already have a CM that holds the environment variables for our containers? If not, do we want to move the env from `spec.container[I].env` to a CM? If yes, this may be a follow-up PR

Reply:
> Do we currently use EnvFrom?

No 

> Do we already have a CM that holds the environment variables for our containers?

No. There are a few environment variables directly set in the `container.env`.

> If not, do we want to move the env from spec.container[I].env to a CM? If yes, this may be a follow-up PR

No. The `EnvFrom` and `Env` here allow our users to set environment variables they want in the container. One purpose for exposing them is to let the outside controllers, like our cloud control plane, set some RW-specific parameters.

Reply:
e.g.,

```yaml
apiVersion: v1
kind: Secret
metadata:
  name: rw-parameters
stringData:
  ETCD_USERNAME: username
  ETCD_PASSWORD: passworde
---
# ...
envFrom:
- prefix: RW_
  secretRef: 
    name: rw-parameters
```